bazel/_gevent_test_main.py

```diff
@@ -38,8 +38,14 @@ def loadTestsFromNames(self, names: Sequence[str], module: str = None) -> unitte
         return self.suite
 
 if __name__ == "__main__":
+    import gevent
     from gevent import monkey
 
+    threadpool = gevent.hub.get_hub().threadpool
+    threadpool.maxsize = 1024
+    threadpool.size = 32
```

Comment:
Some of the tests create hundreds of channels concurrently (which is atypical in production). Without increasing the maximum size of the threadpool, some channels will get starved out, causing the test to take much longer than the native implementation to finish.

This can be removed once the change is made for all channels to share a single completion queue and therefore a single native thread.

Comment added for now.

Reply:
We can also add a public issue with `stale/Never` label